src/components/Book.js

```diff
@@ -0,0 +1,30 @@
+import './styles/Book.css';
+
+function Book() {
+  return (
+    <div className="all-books">
+      <div>
+        <p>Book</p>
+        <h2>Governing the ungovernables</h2>
+        <p>Dr . Ishrat Hussain</p>
+        <p>
+          <span>Edit</span>
+          ||
+          <span>Remove</span>
+          ||
+          <span>comment</span>
+        </p>
```

Comment:
- [ ] Kindly consider taking a `<button>` element here as it is a project requirement. obviously the button won't have the functionality of removing book yet but it needs to be a button element instead of a span text.
   -  Contains individual book state, as well as a button to delete the book

Reply:
For sure, the button would be better than a span as it is a project requirement as well, thank you for your progressive feedback @Zuhaib042